stdlib/src/collections/counter.mojo

```diff
@@ -11,14 +11,11 @@
 # limitations under the License.
 # ===----------------------------------------------------------------------=== #
 
-from collections.dict import Dict
+from collections.dict import Dict, _DictKeyIter, _DictValueIter, _DictEntryIter
 
 
-struct Counter[V: KeyElement](
-    #    Sized,
-    #    CollectionElement,
-    #    Boolable
-):
+@value
```

Comment:
**Request** Since you're significantly expanding the number of methods on this type, I think it would be good for readability if we added some of the standard header comments to visually separate and group different kinds of methods: https://github.com/modularml/mojo/blob/main/stdlib/docs/style-guide.md#code-header-comments

The goal is to have a conventional grouping and ordering for methods so readers can "pattern match" when reading different type definitions and have a good sense at glance about the semantics and way a type is meant to be used.

You can see how those headers are used in most of the other prominent standard library types as well🙂

Reply:
Sure, done here: https://github.com/msaelices/mojo/commit/cfc97a6e09f5ae934d913d3118028f5eaf6e1c64